systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -48,9 +48,15 @@ public static void waitForReconciliation(String testClass, String testName, Stri
         LOGGER.info("Waiting for reconciliation");
         String reconciliation = timeMeasuringSystem.startOperation(Operation.NEXT_RECONCILIATION);
         TestUtils.waitFor("Wait till another rolling update starts", Constants.CO_OPERATION_TIMEOUT_POLL, Constants.RECONCILIATION_INTERVAL + 30000,
-            () -> !cmdKubeClient().searchInLog("deploy", "strimzi-cluster-operator",
+            () -> {
+                LOGGER.info("Logging reconciliation for time {}", timeMeasuringSystem.getCurrentDuration(testClass, testName, reconciliation));
+                LOGGER.info("Reconciliation log:\n{}", cmdKubeClient().searchInLog("deploy", "strimzi-cluster-operator",
```

Comment:
Do we really need this log every poll interval? This operation use executor and `oc logs` command so it's very slow. Maybe rewrite it with some kubernetes watchers will be better, but till that I would remove this log.